Uncaught Error: ENOENT, no such file or directory  C: Users Roland AppData Local atom app-0.187.0 rustc

Enter steps to reproduce below:  1. ... 2. ... **Atom Version**: 0.187.0 **System**: ROLAND-WIN7-PC **Thrown From**:  linter  https://github.com/AtomLinter/Linter  package, v0.12.0 ### Stack Trace Uncaught Error: ENOENT, no such file or directory  C: Users Roland AppData Local atom app-0.187.0 rustc      At fs.js:807 Error: ENOENT, no such file or directory  C: Users Roland AppData Local atom app-0.187.0 rustc  at Error  native  at fs.statSync  fs.js:807:18  at Object.fs.statSync  ATOM_SHELL_ASAR.js:206:16  at LinterRust.Linter._cachedStatSync  C: Users Roland .atom packages linter lib linter.coffee:69:17  at LinterRust.Linter.getCmdAndArgs  C: Users Roland .atom packages linter lib linter.coffee:86:16  at LinterRust.Linter.lintFile  C: Users Roland .atom packages linter lib linter.coffee:127:24  at LinterRust.lintFile  C: Users Roland .atom packages linter-rust lib linter-rust.coffee:42:7  at LinterRust.lintFile  C: Users Roland .atom packages linter-rust lib linter-rust.coffee:1:1  at C: Users Roland .atom packages linter lib linter-view.coffee:158:18 at Array.forEach  native  at C: Users Roland .atom packages linter lib linter-view.coffee:157:18 at FSReqWrap.ONCOMPLETE  fs.js:100:15      ### Commands     -0:16.4 application:OPEN-folder  atom-text-editor.editor  3x -0:08.8 core:backspace  atom-text-editor.editor.is-focused  3x -0:08.1 core:save  atom-text-editor.editor.is-focused      ### Config    json { core : {}, linter : {} }     ### Installed Packages    coffee # User keyboard-localization, v1.1.6 language-rust, v0.4.2 linter, v0.12.0 linter-rust, v0.0.5 minimap, v4.6.0 # Dev No dev packages

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "author": "AtomLinter",
   "version": "0.13.0",
   "description": "A Base Linter with Cow Powers",
-  "repository": "https://github.com/AtomLinter/linter-plus",
+  "repository": "https://github.com/AtomLinter/Linter",
   "license": "MIT",
   "scripts": {
     "lint": "coffeelint lib && eslint lib"

